### ISSUE TITLE
gitstatusd incorrectly reporting an untracked file

### ISSUE THREAD

**Issue by benweissmann** [romkatv/gitstatus#31]
I have a repo that has no untracked files, but `gitstatusd` is reporting one. I've tried restarting `gitstatusd`, and running `git clean -fdX` to clean all ignored files.

<img width="417" alt="Screen Shot 2019-05-25 at 11 44 33 AM" src="https://user-images.githubusercontent.com/207390/58371798-a22bd100-7ee2-11e9-8b5d-2259838fc1ee.png">

Is there some way to see what the file is, or debug the issue?

**Comment by romkatv** [comment]
Please run the following command in the tulip-repo's directory and post its output.

```zsh
zsh -dfxic 'GITSTATUS_ENABLE_LOGGING=1 && source ~/gitstatus/gitstatus.plugin.zsh && gitstatus_start MY && gitstatus_query MY && cat $GITSTATUS_DAEMON_LOG_MY'
```

(You'll probably need to change the path to gitstatus.)

**Comment by benweissmann** [comment]
Thanks! I ran that, and it reported:

```
[2019-05-26 22:44:52 INFO src/repo.cc:178] Found untracked file: tools/dev-env-cookbook/
```

The directory `tools/dev-env-cookbook/` used to have files in it, but they were deleted recently. So the directory still exists on my machine (but has no tracked files it). Weirdly, there was a `.git` directory in there (i.e. `tools/dev-env-cookbook/.git`. I'm not sure why. I deleted that directory and now the output is correct.

Thanks for the help!
